app/models/user.rb

```diff
@@ -10,11 +12,29 @@ class User < ApplicationRecord
 
   has_secure_password
 
+
   def User.digest string
     cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST : BCrypt::Engine.cost
     BCrypt::Password.create(string, cost: cost)
   end
 
+  def User.new_token
+    SecureRandom.urlsafe_base64
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
```

Comment:
dùng update_attributes để đảm bảo tính toàn vẹn của dữ liệu nhé
update_attribute remember_digest: User.digest(remember_token)